Sources/ForageSDK/Foundation/Network/LDManager.swift

```diff
@@ -241,7 +241,7 @@ class LDManager: LDManagerProtocol {
     }
 
     private func createLDConfig(for environment: Environment) -> LDConfig {
-        LDConfig(mobileKey: getLDMobileKey(environment).rawValue)
+        LDConfig(mobileKey: getLDMobileKey(environment).rawValue, autoEnvAttributes: .enabled)
```

Comment:
This is the only code change. LaunchDarkly v9.x requires this second argument to `LDConfig` to get additional environment attributes. I went with their example (passing `.enabled`), but we can also do `.disabled` if we don't want this. More details in the [8.x->9.0 migration guide](https://docs.launchdarkly.com/sdk/client-side/ios/migration-8-to-9?_gl=1%2A1pcqasx%2A_gcl_au%2ANTY3Mjg4ODU3LjE3MTE3NDc5Nzk.).

Reply:
Seems harmless enough and something we may find useful in the future. I say we keep it.